lib/sorbet-rails/gem_plugins/gem_plugins.rb

```diff
@@ -0,0 +1,14 @@
+# typed: true
+require('sorbet-rails/config')
+
+module SorbetRails::GemPlugins
+  extend T::Sig
+
+  sig { void }
+  def self.register_gem_plugins
+    if SorbetRails.config.enabled_gem_plugins.include?(:kaminari)
```

Comment:
I should probably add a check here for whether the configured `enable_gem_plugins` includes only valid symbols, and error if there are any invalid symbols. Probably a better UX than doing nothing and never telling the user they've screwed up.

Reply:
we can do a switch:
```
SorbetRails.config.enabled_gem_plugins.each do |plugin_sym|
  case plugin_sym
  when :kaminari
     ...
  when :pg_search
    ...
  else
    InvalidPlugin.new
end
```